README.md

```diff
@@ -109,6 +109,20 @@ All other attributes are directly converted to custom properties.
 
 For Requests the attributes `http.method` and `http.route` override the Name.
 
+#### Automatic Attribute Mappings
+
+| OpenTelemetry Attribute                           | AppInsights Tag          | Notes                                                                                               |
+|---------------------------------------------------|--------------------------|-----------------------------------------------------------------------------------------------------|
+| Span Id                                           | `ai.operation.id`        |                                                                                                     |
+| Parent Span Id                                    | `ai.operation.parentId`  |                                                                                                     |
+| `SpanKind::Server` + `http.method` + `http.route` | `ai.operation.name`      | The name of the operation becomes `METHOD /the/route`.                                              |
+| `enduser.id`                                      | `ai.user.authUserId`     |                                                                                                     |
+| `service.name` + `service.namespace`              | `ai.cloud.role`          | The role becomes `NAMESPACE.NAME`.                                                                  |
+| `service.instance.id`                             | `ai.cloud.roleInstance`  |                                                                                                     |
+| `service.version`                                 | `ai.application.ver`     |                                                                                                     |
+| `telemetry.sdk.name`                              | `ai.internal.sdkVersion` |                                                                                                     |
+| `ai.*`                                            | `ai.*`                   | All other OpenTelemetry attributes that begin with `ai.` are copied into the AppInsights tags bag.  |
```

Comment:
Two points here:

1. Some of this is already mentioned in the table above. I just referred to the fields as "Context: Application version" instead of "ai.application.ver". It probably makes sense to merge these tables.
1. I don't think this is documented anywhere, which is my fault. The readme is actually generated using [cargo-readme](https://github.com/livioribeiro/cargo-readme) from the lib.rs file. Can you add the documentation to lib.rs instead? That way it's also going to be visible on docs.rs.

Reply:
Sure: honestly, I have not used cargo readme.  Do I need to run it locally and check in the result?

Reply:
Yeah. You run cargo readme -o README.md locally. But don't worry about that. I run that before every release anyway.